src/cmdc_tools/datasets/official/MI/data.py

```diff
@@ -0,0 +1,56 @@
+import asyncio
+
+import pandas as pd
+import pyppeteer
+
+from ...base import DatasetBaseNoDate
+from ..base import CountyData
+
+
+class Michigan(DatasetBaseNoDate, CountyData):
+    def get(self):
+        urls = self._get_urls()
+        cases = self._get_cases(urls[0])
+
+        return cases
+
+    def _get_urls(self):
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
+        urls = loop.run_until_complete(self._get_urls_async())
```

Comment:
We are running on python 3.7+

Let's just do `urls = asyncio.run(self._get_urls_async())`

In python 3.7 the internal Python team exposed that function and now they handle setting up/closing down the event loop ([ref](https://docs.python.org/3/library/asyncio-task.html#asyncio.run))

Reply:
Updated